### DISCUSSION TITLE
Creating an issue from a task item is too easy to do by accident

### DISCUSSION THREAD

**Discussion by RikkiGibson** [community/community#4310] [Projects and Issues]
Just now, I was looking for the "..." menu that lets me edit the issue description, but I accidentally clicked the "circle" button that converts a task item within the issue to a new issue. It was rather inconvenient as I was suddenly stuck with a new issue that I didn't want. It feels like a modal flow similar to "Reference in new issue" for comments might be appropriate.

**Comment by alexisLefebvre** [comment]
Can you still do it? I have the opposite problem, I want to create an issue from a task item but it looks like it only works from a comment: https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-comment
